docs/howto_guides/api_gui.md

```diff
@@ -64,12 +64,12 @@ To retrieve metadata about NMDC data objects (such as files, records, or omics d
 If the data object identifier is known, the metadata can be retrieved using the `GET /data_objects/{data_object_id}` endpoint. Note that only one data object metadata record may be retrieved at a time using this method.<br/>
 <br/>
 
-![find get activities](../_static/images/howto_guides/api_gui/find_get_activities.png)
-The `GET /activities` endpoint is a general way to fetch metadata about various activities (e.g. metagenome assembly, natural organic matter analysis, library preparation, etc.). Any "slot" (a.k.a. attribute) for [WorkflowExecutionActivity](https://microbiomedata.github.io/nmdc-schema/WorkflowExecutionActivity/) or [PlannedProcess](https://microbiomedata.github.io/nmdc-schema/PlannedProcess/) classes may be used in the filter and sort parameters, including attributes of subclasses of `WorkflowExecutionActivity` and `PlannedProcess`. For example, attributes used in subclasses such as, [MetabolomicsAnalysisActivity](https://microbiomedata.github.io/nmdc-schema/MetabolomicsAnalysisActivity/) (subclass of `WorkflowExecutionActivity`) or (Extraction)[https://microbiomedata.github.io/nmdc-schema/Extraction/] (subclass of `PlannedProcess`), can be used as input criteria for the filter and sort parameters of this endpoint.<br/>
+![find get planned processes](../_static/images/howto_guides/api_gui/find_get_planned_process.png)
+The `GET /planned_processes` endpoint is a general way to fetch metadata about various planned processes (e.g. mass spectrometry, natural organic matter analysis, library preparation, etc.). Any "slot" (a.k.a. attribute) for [PlannedProcess](https://microbiomedata.github.io/nmdc-schema/PlannedProcess/) subclasses may be used in the filter and sort parameters. For example, attributes used in subclasses such as, [MetabolomicsAnalysis](https://microbiomedata.github.io/nmdc-schema/MetabolomicsAnalysis/) (subclass of `WorkflowExecution` which is a subclass of `PlannedProcess`) or (Extraction)[https://microbiomedata.github.io/nmdc-schema/Extraction/] (subclass of `MaterialProcessing` which is a subclass of `PlannedProcess`), can be used as input criteria for the filter and sort parameters of this endpoint.<br/>
 <br/>
 
-![find get activities by activity id](../_static/images/howto_guides/api_gui/find_get_activities_activity_id.png)
-If the activity identifier is known, the activity metadata can be retrieved using the `GET /activities/activity_id` endpoint. Note that only one metadata record for an activity may be returned at a time using this method.<br/>
+![find get planned processes by planned process id](../_static/images/howto_guides/api_gui/find_get_planned_process_id.png)
+If the planned process identifier is known, the metadata can be retrieved using the `GET /planned_processes/planned_process_id` endpoint. Note that only one metadata record for a planned process may be returned at a time using this method.<br/>
```

Comment:
I suggest including the curly braces as a way of distinguishing the constant parts of the path from the variable parts of the path:

```suggestion
If the planned process identifier is known, the metadata can be retrieved using the `GET /planned_processes/{planned_process_id}` endpoint. Note that only one metadata record for a planned process may be returned at a time using this method.<br/>
```